packages/extension-ui/src/Popup/Accounts/Account.tsx

```diff
@@ -75,7 +75,13 @@ function Account ({ address, className, genesisHash, isExternal, isHardware, isH
         href={`https://uncoverexplorer.com/account/${address}`}
         target="_blank"
       >
-        {t<string>('View on UNcover')}
+        {t<string>('View on UNcover (Azalea)')}
```

Comment:
is there some way to use the network select dropdown to switch this instead?

<img width="219" alt="Screen Shot 2021-08-03 at 17 10 19" src="https://user-images.githubusercontent.com/5133901/127961192-3f248e8e-a250-47ea-80a9-b59011c818f3.png">

Reply:
that's a good idea... I will look at this.. as currently I can see
<img width="535" alt="Screen Shot 2021-08-04 at 9 19 16 AM" src="https://user-images.githubusercontent.com/29415595/128087681-428ef92f-9292-43c0-b6bf-775c94d76c82.png">
and when user selects CENNZnet Nikau I can redirect it to Nikau uncover else it will always be Azalea uncover